todo-facebookoauth/FacebookOAuth/www/scripts/facebookSignIn.js

```diff
@@ -0,0 +1,42 @@
+﻿var app = {
+
+    login: function () {
+        if (!window.cordova) {
+            var appId = prompt("Enter FB Application ID", "");
+            facebookConnectPlugin.browserInit(appId);
```

Comment:
Yes, when using the Facebook SDK for JavaScript it declares the FB global variable that we use to interact with Facebook. In the case of this plugin it defines this facebookConnectPlugin global variable (see https://github.com/Wizcorp/phonegap-facebook-plugin/blob/master/facebookConnectPlugin.js) for us to interact with the plugin.
I could assign this to a global FB variable to follow the facebook SDK standard but that could conflict with someone trying to use this code in a project where the Facebook SDK is also referenced.